examples/customHttpClient/main.go

```diff
@@ -11,7 +11,7 @@ import (
 
 func main() {
 
-	tmdbClient, err := tmdb.Init(os.Getenv("APIKey"))
+	tmdbClient, err := tmdb.Init(os.Getenv("APIKey"), os.Getenv("SessionID"))
```

Comment:
Hi, this new parameter here will break the current implementation when someone update the lib. Perharps a new function is a better alternatine, like "setSessionID()". What do you think?

Reply:
yeah .. since POST and DELETE needs sessionid .. how about these methods take-in a sessionid ?

Reply:
```
(c *Client) SetSessionID(sid string) error { 
  if sid == "" { 
    return errors.New("The SessionID is empty")
  }
  c.sessionID = sid
}

import "github.com/cyruzin/golang-tmdb"

tmdbClient, err := tmdb.Init(os.GetEnv("TMDB_KEY")

tmdbClient.SetAutoRetry()

// Inside the POST and DELETE
c.SetSessionID("SESSION_ID")
if err != nil {
    fmt.Println(err)
}
```

Reply:
c.SetSessionID("SESSION_ID") - in here SESSION_ID is env variable ?

Reply:
Yes :) 

```
c.SetSessionID(os.GetEnv("SESSION_ID"))
if err != nil {
    fmt.Println(err)
}
```

Reply:
I was thinking, this way, we are going to force the users to use ENV variables, though is more secure thay way. You could move this function near the initialization, like this:

tmdbClient, err := tmdb.Init(os.Getenv("APIKey"))
if err != nil {
 log.Println(err)
}

tmdbClient.SetSessionID(os.GetEnv("SESSION_ID"))

Reply:
will send new PR